screenpy_selenium/questions/element.py

```diff
@@ -32,8 +32,10 @@ def answered_by(self, the_actor: Actor) -> Optional[WebElement]:
         """Direct the Actor to find the element."""
         try:
             return self.target.found_by(the_actor)
-        except TargetingError:
+        except TargetingError as exc:
+            self.caught_exception: Optional[Any] = exc
```

Comment:
I'm curious about what `mypy` said about this line. `Optional[Any]` is very weird to me. 

Is this because of the typing in `screenpy`'s protocol? I left a comment over there that we should probably have that typed as `Exception` instead.

Reply:
Agreed.  This has been updated